From 5d766a2966c71c74e178e356b433b77cba0595d3 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Thu, 22 Mar 2018 02:14:53 +0200
Subject: [PATCH 02/17] legacy: Remove wfdservice_service policy

* Its definition has been moved to private
  and we don't really need it

Change-Id: Iac46354f3210c4ef7997ffb309aba3c3534b340e
---
 legacy-common/mediacodec.te | 1 -
 1 file changed, 1 deletion(-)

diff --git a/legacy-common/mediacodec.te b/legacy-common/mediacodec.te
index 399d77b..b64f06e 100644
--- a/legacy-common/mediacodec.te
+++ b/legacy-common/mediacodec.te
@@ -1,3 +1,2 @@
 # Allow binder communication
-allow mediacodec wfdservice_service:service_manager find;
 allow mediacodec audioserver_service:service_manager find;
-- 
2.7.4

